snuba/datasets/configuration/issues/storages/search_issues.yaml

```diff
@@ -128,3 +128,10 @@ stream_loader:
   default_topic: generic-events
   commit_log_topic: snuba-generic-events-commit-log
   dlq_topic: snuba-dead-letter-generic-events
+
+deletion_processors:
+  - processor: ColumnFilterProcessor
+    args:
+      column_filters:
+        - project_id
+        - replay_id
```

Comment:
this is for search issues so it won't be `replay_id` anymore

Reply:
We will add occurrence_id